docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -510,15 +510,42 @@ Row level deletion
 Tables using v2 of the Iceberg specification support deletion of individual rows
 by writing position delete files.
 
-Rolling back to a previous snapshot
------------------------------------
+Snapshots
+---------
 
 Iceberg supports a "snapshot" model of data, where table snapshots are
-identified by an snapshot IDs.
+identified by a snapshot ID.
+
+The connector provides a system table containing snapshot information for every
+Iceberg table. Snapshots are identified by ``BIGINT`` snapshot IDs.
+You can find the snapshot IDs for table ``customer_orders`` by running the
+following command::
+
+    SELECT snapshot_id FROM iceberg.testdb."customer_orders$snapshots" ORDER BY committed_at DESC
+
+Time travel queries
```

Comment:
I feel like we'd probably want to label this "querying historical data" or something to that effect. "Time travel" sounds a lot more exciting, but it's less precise. I think we've generally avoided using the term time travel in release notes and user-facing docs/communication.

Reply:
"Time travel" is the term that got established for this kind of functionality.
If we look at Iceberg , Delta , Hudi , Snowflake on their website - everywhere is used this term for this kind of functionality.